examples/clip/qdq/laion_clip_text_b32_qdq.json

```
@@ -0,0 +1,103 @@
+{
+    "input_model": {
+        "type": "PytorchModel",
+        "model_path": "laion/clip-vit-b-32-laion2b-s34b-b79k",
+        "generative": false,
+        "io_config": {
+            "input_names": [ "input_ids", "attention_mask" ],
+            "input_shapes": [ [ 1, 77 ], [ 1, 1, 77, 77 ] ],
```

Is the custom loader added to supported 4d mask and improve accuracy? We are expecting a PR soon which does some updates on the update attention mask surgery and makes it work for clip models as well. This is the commit if you would like to try it https://github.com/microsoft/Olive/commit/88a6a0b24f16a818725a3d957fd29e26b218a413. With this change, all ops can be quantized. 

If the accuracy is good with this change and latency is similar, I think using the default 2d mask and loader like the old examples might be better.

---

#1803